### ISSUE TITLE
Version 2.5.0 breaks implementation with headings 

### ISSUE THREAD

**Issue by remcokalf** [kimyvgy/simple-scrollspy#59]
We are using scrollspy on a sidebar navigation.
The `sectionClass` option is set to the class of the `h2` and `h3` headings in the content.

This was working fine until v2.4.2, but is now broken in v2.5.0.
See the following videos. Watch the red active links in the right sidebar:

**2.4.2**
[https://d.pr/v/h7hzXc](https://d.pr/v/h7hzXc)
The nav item remains active until the next one comes into view.

**2.5.0** 
[https://d.pr/v/N6YXfL](https://d.pr/v/N6YXfL)
The nav item now only remains active very shortly. The active class is removed right after the heading is scrolled by. 

This is unusable for us like this. I guess the fixes in the new implementation now need an actual encompassing section element, which we don't have in the content.

Not sure if you'd consider this a bug or if we were just lucky this worked for headings too before. 

If this cannot be fixed, would it be an idea to add an option to make this work again?
Otherwise we'll need to stay on 2.4.2. 

It's a bit disappointing that this new issue seems to be caused by fixing a bug I added in [https://github.com/kimyvgy/simple-scrollspy/issues/29](https://github.com/kimyvgy/simple-scrollspy/issues/29) in 2021.

**Comment by kimyvgy** [comment]
@remcokalf, thank you for your feedback. I am addressing this issue and have added a [new demo page](https://kimyvgy.github.io/simple-scrollspy/demo/toc.html) to replicate the problem. However, I was unable to reproduce it. :( Could you please provide the link to your page shown in the video https://d.pr/v/N6YXfL? I believe identifying the cause will help in resolving it.

**Comment by remcokalf** [comment]
@kimyvgy Appreciate you looking into this. 
See any deeper [Help Center](https://facetwp.com/help-center/) page on the FacetWP site. Here is a simple one with 5 heading and sticky sidebar so you can see it best:
[https://facetwp.com/help-center/troubleshooting/using-the-right-query/](https://facetwp.com/help-center/troubleshooting/using-the-right-query/).
The 5 links in the right sidebar under 'In this article' have scrollspy on them. Active ones have the red hover color.
These pages use v2.4.2, which works. With 2.5.0 not, as shown in the video.

The differerence with the demo page you linked is that the demo has `<section>` elements that have the anchor `id`. 
Our pages don't have sections but only headings (`h2`, `h3`), with `class` `heading-anchor`. We use this to initialize scrollspy, [here](https://facetwp.com/wp-content/themes/fwp4/assets/js/helpcenter.js):
```
if (document.getElementById('tableofcontents')) {
    if (typeof scrollSpy === "function") {
        scrollSpy('#tableofcontents', {
            sectionClass: '.heading-anchor',
            menuActiveTarget: '.toc-link',
            offset: 35, 
        })
    }
}

```

**Comment by kimyvgy** [comment]
@remcokalf The issue has been resolved in version 2.5.1. Kindly update your application and inform me if any problems persist.

**Comment by remcokalf** [comment]
@kimyvgy Hi, I tested 2.5.2, works fine again, thanks! Looking forward to an alternative fix of [#29](https://github.com/kimyvgy/simple-scrollspy/issues/29).
